Makefile

```diff
@@ -34,6 +34,32 @@
 SHELL := bash
 ENABLE_FEATURES ?=
 
+# Frame pointer is enabled by default.
+#
+# If you want to disable frame-pointer, please manually set the environment
+# variable `TIKV_FRAME_POINTER=0 make` (This will fallback to `libunwind`
+# based stack backtrace.).
+ifndef TIKV_FRAME_POINTER
+export TIKV_FRAME_POINTER=1
+endif
+
+# The Rust standard library is recompiled by default. (The purpose is to enable
+# frame pointers in std)
+#
+# If you want to avoid recompiling the Rust standard library, please manually
+# set the environment variable `TIKV_BUILD_STD=0 make` (this will lose CPU Profiling
+# samples related to Rust std functions).
+ifndef TIKV_BUILD_STD
+export TIKV_BUILD_STD=1
```

Comment:
What's the build time difference?

Reply:
By measuring the execution time of `make dist_release`, on my laptop it took 26 minutes when build-std was off and 28.5 minutes when it was on.

Reply:
> this will lose CPU Profiling samples related to Rust std functions

What does it mean? Will it be truncated or ignored?

Reply:
> > this will lose CPU Profiling samples related to Rust std functions
> 
> What does it mean? Will it be truncated or ignored?

If the function in std does not contain a frame pointer, it means that the location on the stack where the frame pointer should be saved will be unknown. When `pprof-rs` encounters such an illegal address during backtracking, the backtracking will be stopped. That means we will lose a lot of samples and the final flame graph will be distorted.

But it doesn't matter, we just need to close `TIKV_FRAME_POINTER` when closing `TIKV_BUILD_STD`, the backtracking will fallback to `libunwind`.

Reply:
Then why not just use one switch?

Reply:
> Then why not just use one switch?

Although we rarely use one of these options alone, I think behaviorally these are two different things.
Although rare, it is safe to have `TIKV_FRAME_POINTER` on, and `TIKV_BUILD_STD` off, without even specifying `RUSTFLAGS=-Cforce-frame-pointers=yes`. This just gets us a call stack with a depth of 1, and reduces the performance overhead appropriately.

Reply:
Don't over optimize, simple is better.

Reply:
> Don't over optimize, simple is better.

What do you think about using a single `TIKV_FRAME_POINTER` to control both frame-pointer option and build-std option? I can change to this pattern.

Reply:
Updated, all behaviors can be controlled with just `TIKV_FRAME_POINTER` now. cc @BusyJay